### ISSUE TITLE
login.windows.net

### ISSUE THREAD

**Issue by p1r473** [DataMaster-2501/DataMaster-Android-AdBlock-Hosts#19]
Its just a Microsoft/Windows login domain.
Not spyware and not malicious.

**Comment by DataMaster-2501** [comment]
Yes it is a "Microsoft/Windows login domain" but you should not be using that  anyway
because if you look login.windows.net is just doing a http 302 redirect to login.microsoftonline.com
So they don't want you using login.windows.net anyway.
Also windows.net is some hosting cloud shit for websites & also bad rep as well.
Do a google on windows.net & you would see all kinds of crap get hosted by it.
So to recap I will remove login.windows.net but only that one.

**Comment by DataMaster-2501** [comment]
the fix will be in my next release
